doc/source/reference/arrays.classes.rst

```diff
@@ -409,24 +409,27 @@ alias for "matrix "in NumPy.
 
 Example 1: Matrix creation from a string
 
->>> a = np.asmatrix('1 2 3; 4 5 3')
->>> print((a*a.T).I)
+  >>> import numpy as np
```

Comment:
You only need to do this once per `rst` file or example section in a docstring.

Also, to ease reviewing, I think it would be best to only add imports and not make language or other changes in this PR.

Reply:
> You only need to do this once per rst file or example section in a docstring.

Actually, we'll need to do this per block of code, unless we want all Examples 1–3 here under the same `TryExamples` block. I think it depends on the length of the content present between each block. For example, the imports would be good to have just once here, however, in other places where there is a lot of text between code blocks, it makes the examples a tad difficult to follow (especially because of the limited width available inside the notebook layout, plus considering the primary and secondary sidebars take up a lot of space).

> Also, to ease reviewing, I think it would be best to only add imports and not make language or other changes in this PR.

Sure – I can leave the ones that I already included, but will make note of this and won't add any further changes! Please let me know if I should remove them altogether.

Reply:
Oops, you're right. I thought SciPy stuck to one import per rst file, but that's wrong on a second look.

Reply:
Added the SciPy implementation to the PR description as well for other readers